docs/root/version_history/current.rst

```diff
@@ -65,6 +65,7 @@ Minor Behavior Changes
   Windows has been disabled due to suboptimal behavior. See the field documentation for more
   information.
 * listener: destroy per network filter chain stats when a network filter chain is removed during the listener in-place update.
+* lua: added ``getAtIndex()`` and ``getValueSize()`` methods to get the value of a header at certain index and get the value size of a given header.
```

Comment:
nit.
```suggestion
* lua: added ``header:getAtIndex()`` and ``header:getValueSize()`` methods to :ref:`header object wrapper <config_http_filters_lua_header_wrapper>` to get the value of a header at certain index and get the value size of a given header.
```

And I would be thankful if you could help to move this line: `..config_http_filters_lua_header_wrapper` right on top of the 

```
Header object API
-----------------
```

I think that was a bad merge.

Reply:
Done